lib/ansible/modules/network/mlnxos/mlnxos_mlag_vip.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python
+#
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: mlnxos_mlag_vip
+version_added: "2.5"
+author: "Samer Deeb (@samerd)"
+short_description: configures MLAG VIP on mlnxos network devices
+description:
+  - This module provides declarative management of MLAG virtual IPs
+    on MLNX-OS network devices.
+notes:
+  - tested on Mellanox OS 3.6.4000
+options:
```

Comment:
Please ensure that all `descriptions:` are full sentences (capital letters & full stop)